pages/instructorAssessmentDownloads/instructorAssessmentDownloads.sql

```diff
@@ -65,7 +65,9 @@ SELECT
     iq.number_attempts,
     DATE_PART('epoch', iq.duration) AS duration_seconds,
     g.name AS group_name,
-    groups_uid_list(g.id) AS uid_list
+    groups_uid_list(g.id) AS uid_list,
+    COALESCE(agu.name, agu.uid) AS assigned_grader_name,
+    COALESCE(lgu.name, lgu.uid) AS last_grader_name
```

Comment:
It may be useful to return both the name and UID as separate columns? UIDs will be unique, whereas names may not be.

Reply:
I'm trying to avoid crowding the files with too many columns. I can replace them with UIDs only, if that is a concern...

Reply:
In that case, it may be better to just use UIDs - I believe that's what we do for group members (UIDs only, no names).

Reply:
Done in 0204092.